### PR TITLE
Update to play 2.5

### PR DESCRIPTION
**Pull request by DmytroOrlov**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -3,12 +3,13 @@ logLevel := Level.Warn
...
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.0")
 
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.7")
 
-addSbtPlugin("com.vmunier" % "sbt-play-scalajs" % "0.2.9")
+addSbtPlugin("com.vmunier" % "sbt-play-scalajs" % "0.3.0")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-gzip" % "1.0.0")
 
+//addSbtPlugin("com.typesafe.sbt" % "sbt-less" % "1.1.1")
```
Comment (novakov-alexey) [inline project/plugins.sbt]: Do we need this commented code? can you just remove it?

File: server/test/controllers/TaskSolverTest.scala [hunk 2]
```diff
@@ -25,7 +26,7 @@ class TaskSolverTest extends PlaySpec with MockFactory with OneAppPerSuite {
         val description = "some description"
         val template = "some template"
         val replyTask = Task(year, scalaClass, timeuuid, "array", description, template, "ref", "test suite")
-        val taskSolver = new TaskSolver(mock[TestExecutor], dao, new MockMessageApi)
+        val taskSolver = new TaskSolver(mock[TestExecutor], dao, new MockMessageApi, new Provider[play.api.Application] { def get() = app})
```
Comment (novakov-alexey) [inline server/test/controllers/TaskSolverTest.scala]: extract code "new Provider ..." to function and call 3 times.

File: server/test/controllers/application/ApplicationTest.scala [hunk 3]
```diff
@@ -40,7 +40,7 @@ class ApplicationTest extends PlaySpec with OneAppPerSuite {
     }
     "post form with missed fields to addTask" should {
       "result BadRequest with error1" in {
-        val Some(result) = route(FakeRequest(POST, "/addTask").withFormUrlEncodedBody("taskName" -> "0", "taskDescription" -> "1", "solutionTemplate" -> "2", "referenceSolution" -> "3" /*, "suite" -> "4"*/))
+        val Some(result) = route(app, FakeRequest(POST, "/addTask").withFormUrlEncodedBody("taskName" -> "0", "taskDescription" -> "1", "solutionTemplate" -> "2", "referenceSolution" -> "3" /*, "suite" -> "4"*/))
```
Comment (novakov-alexey) [inline server/test/controllers/application/ApplicationTest.scala]: there are some commented code in arguments. Let's get rid of any commented code in master.

Comment (DmytroOrlov) [inline server/test/controllers/application/ApplicationTest.scala]: this is good one

File: server/test/controllers/TaskSolverTest.scala (hunk 2 above)
Comment (DmytroOrlov) [inline server/test/controllers/TaskSolverTest.scala]: i'll remove it later completely

Comment (novakov-alexey) [inline server/test/controllers/TaskSolverTest.scala]: When? Nobody holds us to avoid duplicates even in temporary code.

Comment (DmytroOrlov) [inline server/test/controllers/TaskSolverTest.scala]: whent change this play.api.mvc.WebSocket#acceptWithActor
to play.api.libs.Streams.ActorFlow.actorRef
and than to Sink.actorRef and Source.actorRef

Comment (novakov-alexey) [inline server/test/controllers/TaskSolverTest.scala]: Ok. It will be separate task, right?

Comment (DmytroOrlov) [inline server/test/controllers/TaskSolverTest.scala]: just created

**Pull request merged**